_docs/release-schedule.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Release schedule
+order: 8
+longtitle: What to expect in future releases of Portus
+---
+
+## Release schedule for 2.2
+
+### The dates
+
+- Code freeze: January 10th 2017.
+- Release: January 13th 2017.
+- Patch-level releases will come as needed until a new major/minor release
+  comes.
+
+### What to expect
+
+The 2.2 release is going to be a continuation of what the 2.1 release
+delivered. Thus, this release will contain a couple of features that didn't make
+it into the 2.1 release in time (e.g. namespace removal), and some improvements
+to make Portus nicer to use for both users and administrators.
+
+The complete list of features/improvements are listed in
+[this milestone](https://github.com/SUSE/Portus/milestone/16). Some highlights:
+
+- Provide a Docker image that can be used both in small deployments and in more
+  complex scenarios. This is a long running issue that we'd like to finally get
+  done during this release.
+- Allow administrators to remove namespaces. This is something that we wanted to
+  tackle for the 2.1 release but we ran out of time.
+- Improve the experience for openSUSE/SLE users. That is, `portusctl` needs
+  quite some improvements on the UI/UX front (e.g. handle upgrades smoothly).
+
+Besides that, the list of feature/improvements requests keeps on growing. We
```

Comment:
But then it would basically read "features requests and improvements requests" which seems incorrect. We could just drop the "requests" word :wink: